clang-tools-extra/clang-tidy/cppcoreguidelines/MissingStdForwardCheck.cpp

```diff
@@ -86,6 +88,15 @@ AST_MATCHER(VarDecl, hasIdentifier) {
 
 } // namespace
 
+MissingStdForwardCheck::MissingStdForwardCheck(StringRef Name,
+                                               ClangTidyContext *Context)
+    : ClangTidyCheck(Name, Context),
+      IgnoreStaticCasts(Options.get("IgnoreStaticCasts", false)) {}
```

Comment:
The option name sounds too generic, and suggests that users can then use any static cast without a diagnostic being generated. Maybe `IgnoreStaticCastToLValueReference` (or something else)